alchemiscale/storage/statestore.py

```diff
@@ -136,14 +136,15 @@ def _generate_claim_query(
     UNWIND {task_data} AS task_sk
     MATCH (t:Task {{_scoped_key: task_sk}})
     WHERE NOT (t)<-[:CLAIMS]-(:ComputeServiceRegistration)
-    SET t.status = 'running'
 
     WITH t
 
     // create CLAIMS relationship with given compute service
     MATCH (csreg:ComputeServiceRegistration {{identifier: '{compute_service_id}'}})
     CREATE (t)<-[cl:CLAIMS {{claimed: localdatetime('{datetime.utcnow().isoformat()}')}}]-(csreg)
 
+    SET t.status = 'running'
```

Comment:
updating this to use `TastStatusEnum.running.value`

Reply:
Awesome, thank you!